src/components/learning-center/category-menu.tsx

```diff
@@ -1,25 +1,29 @@
 import React from 'react'
 import { StaticQuery, graphql, Link } from 'gatsby'
-import { ContentfulContent } from '../../pages';
+import { ContentfulContent, Locale } from '../../pages';
 import { Category } from '../../pages/learn';
 
 type CategoryMenuProps = 
   ContentfulContent & 
-  {selectedCategory?: any}
+  {selectedCategory?: any} & 
+  Locale
 
-const CategoryMenuScaffolding = (props: CategoryMenuProps) => (
-  <div className="field is-centered is-hidden-mobile">
-    {(props.content.categoryButtons).map( 
-    (category: Category, i: number) =>
-    <Link key={i} to={'/learn/category/' + category.slug} 
-      className={"button is-primary is-uppercase " + 
-        /* Small button size on category pages: */ 
-        (props.selectedCategory ? " is-small" : "") + 
-        /* Highlighted button for selected category: */
-        (props.selectedCategory && props.selectedCategory !== category.slug ? " is-outlined" : "")} >
-      {category.title}
-    </Link>)}
-  </div>)
+const CategoryMenuScaffolding = (props: CategoryMenuProps) => {
+  const localePrefix = props.locale ? ("/" + props.locale) : ""; 
+  return (
+    <div className="field is-centered is-hidden-mobile">
+      {(props.content.categoryButtons).map( 
+      (category: Category, i: number) =>
+      <Link key={i} to={localePrefix + '/learn/category/' + category.slug} 
+        className={"button is-primary is-uppercase " + 
+          /* Small button size on category pages: */ 
+          (props.selectedCategory ? " is-small" : "") + 
+          /* Highlighted button for selected category: */
+          (props.selectedCategory && props.selectedCategory !== category.slug ? " is-outlined" : "")} >
```

Comment:
Have you considered using the [`classnames`](https://github.com/JedWatson/classnames#readme) library here?  It's what I use for tenants2 and it makes defining a bunch of classes declaratively like this a lot more readable (and prevents accidentally forgetting to put a space between class names).

Reply:
oh cool! I'll take a look at that :)